JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -918,7 +919,7 @@ AnovaRepeatedMeasures <- function(dataset=NULL, options, perform="run", callback
 
 						row[["eta"]] <- SS / SSt
 						row[["partialEta"]] <- SS / (SS + SSr)
-						omega <- (SS - (df * MSr)) / (SSt + MSr)
+						omega <- (SS - (df * MSr)) / (SSt + MSr) #p480 Field
```

Comment:
This still needs changing, doesn't it?

Reply:
Updated in latest commit! Maybe @Kucharssim can recheck the calculation now?